app/paypal.py

```
@@ -0,0 +1,230 @@
+                                ])
+                                sql_connection.cursor().execute("UPDATE chat_users.users SET additional_remaining_messages=additional_remaining_messages+100 WHERE user_id=?", [user_id])
+                            elif item['item_code'] == PAYPAL_ITEMCODE_RECHARGEFREQUENCY:
+                                log_info('FOUND RECHARGEFREQUENCY in transaction')
+                                sql_connection.cursor().execute("INSERT INTO purchases (user_id, at_datetime, amount, product, paypal_transaction_id) VALUES (?, ? ,?, ?, ?)", [
+                                    user_id,
+                                    transaction['transaction_info']['transaction_initiation_date'],
+                                    item['item_quantity'] or 1,
+                                    'RECHARGEFREQUENCY',
+                                    transaction['transaction_info']['transaction_id']
+                                ])
+                                for i in range(item['item_quantity']):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)